x-pack/plugins/task_manager/server/create_task_manager.test.ts

```diff
@@ -39,20 +39,21 @@ describe('createTaskManager', () => {
     const mockLegacyDeps = getMockLegacyDeps();
     const setupResult = createTaskManager(mockCoreSetup, mockLegacyDeps);
     expect(setupResult).toMatchInlineSnapshot(`
-          TaskManager {
-            "addMiddleware": [MockFunction],
-            "assertUninitialized": [MockFunction],
-            "attemptToRun": [MockFunction],
-            "ensureScheduled": [MockFunction],
-            "fetch": [MockFunction],
-            "registerTaskDefinitions": [MockFunction],
-            "remove": [MockFunction],
-            "runNow": [MockFunction],
-            "schedule": [MockFunction],
-            "start": [MockFunction],
-            "stop": [MockFunction],
-            "waitUntilStarted": [MockFunction],
-          }
-        `);
+      TaskManager {
```

Comment:
I'm curious if this test is even useful.  It doesn't bring me joy :-)

Reply:
That's a fair question, it's left overs from Legacy IMHO, but until we migrate everything to TS and NP, feels like this is still worth keeping around?